### PR TITLE
Update DIGISIGN.ID member profile

### PR DESCRIPTION
**Pull request by budi-rahman**
update data digisign

### CODE REVIEW

File: data/members/digisign-id.yaml [hunk 1]
```diff
@@ -4,19 +4,37 @@ slogan: We delivered Digital Signature, Digital Certification, Dekstop Signing,
 website: https://digisign.id/
 
 # Short description, longer content can be placed in `content/members/`
-description: Digisign is a brand of PT. Net Internusa solutions are specialized in Electronic Signatures (Digital Signature) & Electronic Certificate Providers (CA-Certificate Authority).
+description: Digisign is a brand of PT. Solusi Net Internusa are specialized in Electronic Signatures (Digital Signature) & Electronic Certificate Providers (CA-Certificate Authority).
 
 # membership
 memberSince: 2023-05-01
-memberType: F
+memberType: CA
```
Comment (vanbroup) [inline data/members/digisign-id.yaml:11]: The application of digisign.id is done under category F not A, can you please provide a link to your repository, audits and in which root stores you have been included?

File: data/members/digisign-id.yaml [hunk 2]
```diff
@@ -4,19 +4,37 @@ slogan: We delivered Digital Signature, Digital Certification, Dekstop Signing,
...
+      
+  - name: Aa Iksan Aripin
+    role: Chief Technologi Officer
+    social:
+      linkedin: https://www.linkedin.com/in/aaiksan/
+
+  - name: Pungki Arianto
+    role: Digital Certificate Manager
+    social:
+      linkedin: https://www.linkedin.com/in/pungkiarianto/
```
Comment (vanbroup) [inline data/members/digisign-id.yaml:40]: Please add a new line at the end

File: assets/images/members/digisign-id/digisign-id.svg
Comment (vanbroup) [inline assets/images/members/digisign-id/digisign-id.svg:1]: This looks like a traced PNG file with a bad result, please provide a high quality vector, for example by saving a .ai file to .svg.

**Review by vanbroup — CHANGES_REQUESTED** [review]

**Comment by vanbroup** [comment]
@budi-rahman any updates on the pending comments?

**Comment by svenska-primekey** [comment]
No update in 5 months. Changes made in MR #151

**Pull request closed without merging**